main/src/main/java/com/sedmelluq/discord/lavaplayer/source/nico/NicoAudioTrack.java

```diff
@@ -61,7 +61,7 @@ public void process(LocalAudioTrackExecutor localExecutor) throws Exception {
             log.debug("Starting NicoNico track from URL: {}", playbackUrl);
 
             try (PersistentHttpStream stream = new PersistentHttpStream(httpInterface, new URI(playbackUrl), null)) {
-                long heartbeat = info.get("session").get("keep_method").get("heartbeat").get("lifetime").asLong(120000) - 60000;
+                long heartbeat = info.get("session").get("keep_method").get("heartbeat").get("lifetime").asLong(120000) - 1000;
```

Comment:
Perhaps a larger threshold should be allowed here. Maybe 3 or 5 seconds, in case of connection hiccups.